docs/DeveloperGuide.md

```diff
@@ -237,70 +237,403 @@ _{Explain here how the data archiving feature will be implemented}_
 **Target user profile**:
 
 * has a need to manage a significant number of contacts
+* has a need to manage a meeting schedule
+* has a need to oversee the development of the product
 * prefer desktop apps over other types
 * can type fast
 * prefers typing to mouse interactions
 * is reasonably comfortable using CLI apps
 
-**Value proposition**: manage contacts faster than a typical mouse/GUI driven app
+**Value proposition**: Used in a corporate setting, our application integrates both a meeting schedule and a contact list for the user to 
+manage their product’s development more comprehensively and conveniently.
 
 
 ### User stories
 
-Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unlikely to have) - `*`
+Priorities: 
+* `* * *` - High (must have)
+* `* *` - Medium (nice to have)
+* `*` - Low (unlikely to have) 
+
+| Priority | As a …​                               | I want to …​                     | So that I can…​                                                         |
+| -------- | ---------------------------------------- | --------------------------------------------------------------------------- | ---------------------------------- |
+| **Deliverables** |
+| `* * *`  | Product Manager                          | add deliverables                                                            | keep track of them                 |
```

Comment:
Noticed you left out type of Product Managers in here and elsewhere (e.g. "Product Manager who needs to be aware of changes in description of deliverables"). I believe we can leave these out for now. Just a note.